assimilation_code/programs/wakeup_filter/wakeup_filter.rst

```diff
@@ -0,0 +1,40 @@
+PROGRAM ``wakeup_filter``
+=========================
+
+Overview
+--------
+
+Small auxiliary program for use in the "async=4" case where the main filter program is an MPI program and the model
+being run with DART is also an MPI program. The main MPI job script runs each of the model advances for the ensemble
+members, and then runs this program to restart the filter program.
```

Comment:
attention would be good.  the current version can't advance the model inside filter unless you're using combination/single netcdf files (all copies in a single file) for i/o.  so most cases that needed async=4 can't run anyway.

just for the record: at one point i made the code that writes out the state and reads it back in during a model advance work again for netcdf i/o.  but i don't believe that made it into the repo because there were questions about how to name the files or something.

Reply:
Done ... but I'd like to get the 'skunkworks' code Nancy has squirreled away!

Reply:
i promise to look for it.  it's a version of obs_model_mod.f90 with working file code in the advance_state() subroutine.  (i wish i could remember what user i made it for.)